DataFormats/BTauReco/interface/PixelClusterTagInfo.h

```diff
@@ -0,0 +1,69 @@
+#ifndef DataFormats_BTauReco_PixelClusterTagInfo_h
+#define DataFormats_BTauReco_PixelClusterTagInfo_h
+
+#include "DataFormats/BTauReco/interface/BaseTagInfo.h"
+
+namespace reco {
+
+  struct PixelClusterProperties {
+    float x = 0;
+    float y = 0;
+    float z = 0;
+    int charge = 0;
+    int layer = 0;
+  };
+
+  struct PixelClusterData {
+    std::vector<int8_t> r004;
```

Comment:
it looks like this is used as an array of size 4.
Why was std::array not used?

Reply:
ah, nm. there is also an explicit support for phase-0, aiming to save a few chars.